JavaScript/JavaScriptSDK/Telemetry/Common/DataSanitizer.ts

```diff
@@ -77,7 +77,7 @@ module Microsoft.ApplicationInsights.Telemetry.Common {
             if (value) {
                 value = Util.trim(value);
                 if (value.toString().length > DataSanitizer.MAX_STRING_LENGTH) {
-                    value = value.substring(0, DataSanitizer.MAX_STRING_LENGTH);
+                    value = value.toString().substring(0, DataSanitizer.MAX_STRING_LENGTH);
```

Comment:
Looks like there's no difference between substr and substring in the way they behave here, so I'm going to leave it.